app/features-json/setup_json_controller.rb

```diff
@@ -8,5 +8,109 @@ class SetupJSONController < APIController
     get "#{HOME}/configured", authenticate: false do
       return json(Services.onboarding_service.correct_setup?)
     end
+
+    # TODO: we need to protect this API endpoint
+    # one the onboarding user flow, we don't have any accounts yet
+    # however, we need to prohibit sending a POST request to this API endpoint
+    # after CI was already set up
+    post HOME.to_s, authenticate: false do
+      # we set this as default so that we don't need duplicate code for error detection
+      params["bot_account"] ||= {}
+
+      encryption_key = params["encryption_key"]
+      bot_token = params["bot_account"]["token"]
+      bot_password = params["bot_account"]["password"] # TODO: What do we need this for
+      config_repo = params["config_repo"]
+      initial_onboarding_token = params["initial_onboarding_token"]
```

Comment:
I never understood what the initial onboarding token was. Why don't we just use the bot token?

Reply:
I think you're right, we could do that 👍 Will update the existing code base also to reflect this. Anything we're missing @taquitos